copy/copy.go

```diff
@@ -157,6 +158,19 @@ func Image(policyContext *signature.PolicyContext, destRef, srcRef types.ImageRe
 		}
 	}()
 
+	if !dest.AcceptsAnyOSSpecificImage() {
+		c, err := src.OCIConfig()
+		if err != nil {
+			return err
```

Comment:
Consider 
```go
errors.Wrapf("Error parsing image configuration", err)
```
so that the caller doesn’t get a context-less `invalid character … looking for beginning of value` or whatever.